tools/firtool/firtool.cpp

```diff
@@ -110,9 +110,15 @@ static cl::opt<bool>
                            cl::desc("Convert all chisel asserts into SVA"),
                            cl::init(false), cl::cat(mainCategory));
 
-static cl::opt<bool> stripMuxPragmas("strip-mux-pragmas",
-                                     cl::desc("Don't emit mux pragmas"),
-                                     cl::init(false), cl::cat(mainCategory));
+static cl::opt<bool> addMuxPragmas("add-mux-pragmas",
+                                   cl::desc("Annotate mux pragmas"),
+                                   cl::init(false), cl::cat(mainCategory));
+
+static cl::opt<bool> stripMuxPragmas(
+    "strip-mux-pragmas",
+    cl::desc("Strip mux pragmas. This options was deprecated since annotate "
```

Comment:
For your consideration:
```suggestion
    cl::desc("Strip mux pragmas. This option was deprecated since mux pragma annotatations are not...");
```
(modulo wrapping/formatting)